test/test-utils/call.ts

```diff
@@ -68,6 +68,10 @@ export class MockedCall extends Call {
         super.participants = value;
     }
 
+    public setConnectionState(value: ConnectionState) {
+        super.connectionState = value;
+    }
```

Comment:
Why was this added?

Reply:
For testing the Leave button. I need to easily set the connection state. (`call.connect()` raised issues)

Reply:
not needed anymore not